searchlib/src/vespa/searchlib/attribute/postinglistsearchcontext.h

```diff
@@ -300,6 +290,18 @@ StringPostingSearchContext(QueryTermSimpleUP qTerm, bool useBitVector, const Att
     }
 }
 
+template <typename BaseSC, typename AttrT, typename DataT>
+bool
+StringPostingSearchContext<BaseSC, AttrT, DataT>::useThis(const PostingListSearchContext::DictionaryConstIterator & it) const {
+    if ( this->isCased() ) {
+        this->isMatch(_enumStore.get_value(it.getKey()));
```

Comment:
What about cased regex ?
Also, the return value from this->isMatch() is ignored.

Reply:
Planned to take regex in separate round, but after checking code regex case sensitivity is handler construction time.
So reordering the conditions should take care of this.